ocf_datapipes/batch/merge_numpy_modalities.py

```diff
@@ -1,8 +1,9 @@
 """Merge multiple modalities together in NumpyBatch"""
 from torchdata.datapipes import functional_datapipe
-from torchdata.datapipes.iter import IterDataPipe, Zipper
+from torchdata.datapipes.iter import IterDataPipe
 
 from ocf_datapipes.utils.consts import NumpyBatch
+from ocf_datapipes.utils.utils import ZipperIterDataPipe as Zipper
```

Comment:
Could you potentially put this in either `ocf_datapipes.__init__.py` for the init file for utils? So then we can do `from ocf_datapipes import Zipper` instead? To match more of the other datapipes.

Reply:
yea thats a good idea

Reply:
the others are `from ocf_datapipes.select import DropGSP` so ill go with from `ocf_datapipes.utils import Zipper`, if htats ok?